rke2-cloud-provider.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: rke2-cloud-provider
+  version: "1.33.1"
+  epoch: 0
+  description: cloud controller manager for rke2
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - clang
+      - file
+      - gawk
+      - gcc
+      - git
+      - jq
+      - lld
+      - make
```

Comment:
Are any of these needed in the build env? It looks like the pipeline is just using go